### PR TITLE
OSDOCS-9014: docs scrub for kuryr EOL

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9014

Link to docs preview:
https://69170--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_openstack/preparing-to-install-on-openstack

QE review:
- [x] QE has approved this change.

Additional information:
This removes the two main assembly files and xrefs to them. I will continue these efforts in follow up PRs and link back. 
PTAL @dulek

### CODE REVIEW

**Review by lahinson — COMMENTED** [review]
@JoeAldinger Nice work on going through the docs to scrub references to kuryr! I left a couple of comments for small things to fix. I assume this will require change management because it's related to the removal public documentation [1]. Is that your understanding, too?

[1] https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#bookmark=id.khm4rxl210qm

File: installing/installing-preparing.adoc [hunk 1]
```diff
@@ -61,13 +61,13 @@ If you need to perform basic configuration for your installer-provisioned infras
...
...lling_ibm_z/installing-ibm-z.adoc#installing-ibm-z[{ibm-z-name} or {ibm-linuxone-name}], xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[{ibm-z-name} and {ibm-linuxone-name} with {op-system-base} KVM], xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power[], or xref:../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[vSphere], use the specific installation instructions to deploy your cluster. If you use other supported hardware, follow the xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal installation] procedure. For some of these platforms, such as xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack}], xref:../installing/installing_vsphere/installing-vsphere-network-customizations.adoc#installing-vsphere-network-customizations[vSphere], and xref:../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installing-bare-metal-network-customizations[bare metal], you can also customize additional network parameters during installation.
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply installing/installing-preparing.adoc:70]: Awesome catch this is actually a bug in current docs. I'll fix here and then a different PR for the bug.

**Comment by dulek** [comment]
This looks good from my perspective, though obviously still a lot of content is left.

File: welcome/index.adoc [hunk 2]
```diff
@@ -98,10 +98,9 @@ endif::openshift-origin[]
...
 
-- **Install a cluster on {rh-openstack-first}**: You can install a cluster on
-xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack} with customizations], with xref:../installing/installing_openstack/installing-openstack-installer-kuryr.adoc#installing-openstack-installer-kuryr[network customizations], or xref:../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[on a restricted network] on installer-provisioned infrastructure.
-+
-You can install a cluster on xref:../installing/installing_openstack/installing-openstack-user.adoc#installing-openstack-user[{rh-openstack} with customizations] or xref:../installing/installing_openstack/installing-openstack-user-kuryr.adoc#installing-openstack-user-kuryr[with network customizations] on user-provisioned infrastructure.
+- **Install a cluster on {rh-openstack-first}**: You can install a cluster on {rh-openstack} with customizations on a restricted network on installer-provisioned infrastructure. For more information, see the following documentation:
```
Comment (dulek) [inline welcome/index.adoc:101]: Wait, I think these should be listed as distinct options. We have customizations and we have restricted networks, they can be used together, but doesn't need to.

**Review by dulek — CHANGES_REQUESTED** [review]
Okay, one doubt there.

**Pull request merged**